cdap-unit-test/src/main/java/io/cdap/cdap/test/TestBase.java

```diff
@@ -1125,15 +1124,16 @@ private static CConfiguration createPreviewConf(CConfiguration cConf) {
   }
 
   public static SparkCompat getCurrentSparkCompat() {
-    String sparkVersion = package$.MODULE$.SPARK_VERSION();
-    switch (sparkVersion.charAt(0)) {
-      case '3':
-        return SparkCompat.SPARK3_2_12;
-      case '2':
-        return SparkCompat.SPARK2_2_11;
-      default:
-        throw new IllegalStateException("Spark version " + sparkVersion + " is unknown");
-    }
+    return SparkCompat.SPARK3_2_12;
```

Comment:
Should we go ahead and remove these comments?

Reply:
Ack.